src/Components/UserInput.jsx

```diff
@@ -1,12 +1,32 @@
 import { useState } from "react";
 import PropTypes from "prop-types";
 import DialogBox from "./DialogBox";
+import ToastOption from "./ToastOption";
 
 function UserInput({ handleNewMessage }) {
   const [message, setMessage] = useState("");
   const [toastType, setToastType] = useState("");
   const [isModalOpen, setIsModalOpen] = useState(false);
 
+  const toastVariants = [
```

Comment:
@Meet-Kalani This object will be re-created on each render, move it outside the component